test_helpers/main.go

```diff
@@ -204,11 +231,19 @@ func StartTarantool(startOpts StartOpts) (TarantoolInstance, error) {
 		User:       startOpts.User,
 		Pass:       startOpts.Pass,
 		SkipSchema: true,
+		Transport:  startOpts.PingTransport,
+		Ssl:        startOpts.PingSsl,
 	}
 
 	var i uint
 	for i = 0; i <= startOpts.ConnectRetry; i++ {
-		err = isReady(startOpts.Listen, &opts)
+		var server string
```

Comment:
There is no reason to choose a server inside the cycle on each iteration

Reply:
Agree, the code has been extracted from the loop.